src/javascript/app/pages/trade/analysis.js

```diff
@@ -89,14 +111,63 @@ const TradingAnalysis = (() => {
                 showExplanation();
             }
         }
+        if (current_tab) {
+            const el_to_show = getElementById(current_tab);
+            slideSelector(tab_selector_id, el_to_show);
+            const mobile_tab_header = document.getElementById('tab_mobile_header');
```

Comment:
any reason why we're using `document. getElementById` instead of `getElementById` as above (line 115)? also missing the `el_` in the beginning to make it consistent